shell/edit/fleet.cattle.io.gitrepo.vue

```diff
@@ -648,12 +653,13 @@ export default {
       <h2 v-t="'fleet.gitRepo.resources.label'" />
       <div>
         <Checkbox
-          v-model:value="value.spec.correctDrift.enabled"
+          v-model:value="correctDriftEnabled"
```

Comment:
I couldn't repro this issue, although I didn't provision a rancher system from that commit, tbh.

My question is: Using `v-model` and `@update:value` to set the exact same variable value doesn't yield any problems on the console/UI?

We do either https://github.com/rancher/dashboard/blob/master/pkg/gke/components/AdvancedOptions.vue#L179-L184 or https://github.com/rancher/dashboard/blob/master/pkg/gke/components/AdvancedOptions.vue#L169-L173 but I don't think we should do both 🤔 

The rest LGTM

Reply:
@aalves08 you are right, `@update:value="correctDriftEnabled=$event"` is indeed not needed, since the `v-model:value` will update the value on change